debitpolicy/debitpolicy_test.go

```diff
@@ -0,0 +1 @@
+package debitpolicy
```

Comment:
I know this is empty atm, but tests will go in `debitpolicy_test`, assuming we keep this package. Files with a `package` that ends in `_test` are handled specially by Go, in that they can live in the same directory as a package without the suffix, and they are excluded from any non-test builds.

Reply:
Ok yeah i see that in the account_test.go now.

Reply:
I've renamed that to have the "_test" for now. I'll look at this again after potentially moving the debit policy feature into account package as suggested.